lib/mpl_toolkits/axes_grid1/axes_size.py

```diff
@@ -270,3 +273,27 @@ def __call__(self, renderer):
                        ax.bbox)
               for ax in self._ax_list]
         return max(vl)
+
+
+class _AxesDecorationsSize(_Base):
+    _get_size_map = {
+        "left":   lambda tight_bb, axes_bb: axes_bb.xmin - tight_bb.xmin,
+        "right":  lambda tight_bb, axes_bb: tight_bb.xmax - axes_bb.xmax,
+        "bottom": lambda tight_bb, axes_bb: axes_bb.ymin - tight_bb.ymin,
+        "top":    lambda tight_bb, axes_bb: tight_bb.ymax - axes_bb.ymax,
```

Comment:
To clarify, this is saying that the tight bbox will always be bigger than than the Axes bbox (if you want positive results)?

Reply:
This simply copypasted the logic of GetExtentHelper.  TBH I didn't really check, but I guess this makes sense (as the tightbb will also include "decorations", i.e. labels/ticks/titles/etc.)?

Reply:
Ah right, I forgot about decorations; that seems plausible.